test/tck/configure_feature_files.py

```diff
@@ -0,0 +1,30 @@
+import os
+import tarfile
+import urllib2
```

Comment:
Sorry about the random comment here, I just noted that this line will fail on Python3. urllib2 has been renamed urllib (and slightly restructured) there.